queue_services/entity-filer/src/entity_filer/filing_processors/incorporation_filing.py

```diff
@@ -0,0 +1,71 @@
+# Copyright © 2020 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""File processing rules and actions for the incorporation of a business."""
+from typing import Dict
+
+from flask import Flask
+import requests
+
+from entity_queue_common.service_utils import logger
+from entity_filer.filing_processors import create_office
+from legal_api.models import Business, db
+
+
+def get_next_corp_num(business_type, application: Flask):
+    """Retrieve the next available sequential corp-num from COLIN."""
+    resp = requests.get(f'{application.config["COLIN_API"]}/api/v1/businesses')
+
+    if resp.status_code == 200:
+        new_corpnum = resp.json()['corpNum']
+        if new_corpnum:
+            # TODO: Fix endpoint
+            return business_type + str(new_corpnum)
+    return None
+
+
+def insert_business_info(corp_num: str, business: Business, business_info: Dict):
+    """Format and update the business entity from incorporation filing."""
+    if corp_num and business and business_info:
+        business.identifier = corp_num
+        # TODO: Other properties contained in the NR
+    else:
+        return None
+    return business
```

Comment:
I think this function would be better named "update_" rather than "insert_" business_info, since that's what it's doing - the business record already exists.

Reply:
I was thinking "insert" since, up to this point, there is no actual info in the record, but I can change that to Update

Reply:
should be resolved now